pkg/sql/colexec/fill/fill.go

```diff
@@ -50,7 +50,11 @@ func (fill *Fill) Prepare(proc *process.Process) (err error) {
 
 	f := true
 	for i := len(fill.AggIds) - 1; i >= 0; i-- {
-		if fill.AggIds[i] == function.MAX || fill.AggIds[i] == function.MIN || fill.AggIds[i] == function.SUM || fill.AggIds[i] == function.AVG {
+		if fill.AggIds[i] == function.MAX ||
```

Comment:
Overall this is bad code -- should never test fill.AggIds == MIN etc.    Should always be from aggIds, get a function object, then check the property of the function.    

Basically, if I need to add a new aggregate function, all I should do is to add function def/impl in the function package, never need to change code here.

Reply:
bad code, just delete. I forgot to delete it before